### ISSUE TITLE
[Question] Using extra browsers and devices for specific tests

### ISSUE THREAD

**Issue by Tbhesswebber** [microsoft/playwright#18354]
We're just getting started with Playwright after a few issues with Cypress, so I'm a total newbie - pardon the question if it's been answered elsewhere... We and we have a very small subset of tests (maybe three pages in total) in which we need to test on multiple browsers across desktop, iOS, and android.  I'm curious what the recommended approach would be for this.

All of them fall under a very specific category (shared pages), so my current thinking is to add `@shared` to the describe blocks and then add an option to `config.projects` that leverages `testProject.grep` to filter down to just those tests and has `testProject.name` set to "Shared Pages" (or similar) since they will always require the same support on those pages.

Is this the recommended path? Is there a way to leverage local configuration and a reusable configuration object that might be preferable? What would tradeoffs be?

**Comment by mxschmitt** [comment]
This is the recommended approach, we do the same for our smoke tests, which run e.g. inside Docker, only a subset are marked as `@smoke` and we run them on exotic OS or browser combinations.

TL;DR: Add @xxx to the test title, create a new project, use grep to filter the important tests out and run it via that.

**Comment by Tbhesswebber** [comment]
Perfect! Thanks! Largely, these tests are "did it render without errors", so emulators that modify the UA and engine are more than good enough!
